examples/tutorials/Introduction_to_ProtBERT.ipynb

```diff
@@ -0,0 +1,1114 @@
+{
```

Comment:
<p>Since the pretraining is a key part of the tutorial, can we see a plot of the training ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Added.</p>